src/replace-libraries-ngpackagr-builder.ts

```diff
@@ -0,0 +1,63 @@
+import {versions} from './angular-versions';
+import {AngularJson} from './types/angular-json';
+import {ArchitectConfiguration} from './types/architect-configuration';
+
+export const pre10_1LibraryBuilder = '@angular-devkit/build-ng-packagr:build';
+export const since10_1LibraryBuilder =
```

Comment:
suggestion: Use a name similar to the test data to be precise and consistent, for example something like `angular10_1andUpLibraryBuilder`.